qan-api2/models/metrics.go

```diff
@@ -978,7 +978,7 @@ func (m *Metrics) QueryExists(ctx context.Context, serviceID, query string) (boo
 	return false, nil
 }
 
-const queryByQueryIDTmpl = `SELECT fingerprint, placeholders_count FROM metrics
+const queryByQueryIDTmpl = `SELECT explain_fingerprint, placeholders_count FROM metrics
```

Comment:
That's breaking change, no?

Reply:
I think its not. Explain_fingerprint exists even before (because before data are stored in clickhouse it has explain_fingerprint and fingerprint too) and we converting explain_fingerprint into fingerprint format anyway. Did I miss something?

Reply:
Aha, ok then.